examples/shmem/lidar_obstacles.py

```diff
@@ -0,0 +1,39 @@
+import ctypes
+
+import sysv_ipc
+
+
+class ShmData(ctypes.Structure):
+    _fields_ = [
+        ('tof', ctypes.c_ushort * 6),
+        ('lidar', ctypes.c_ushort * 360)
+    ]
+
+
+shm_ptr = sysv_ipc.SharedMemory(
+    key=None, mode=0o666, flags=sysv_ipc.IPC_CREX,
+    size=ctypes.sizeof(ShmData)
+)
+shm_data = ShmData.from_buffer(shm_ptr)
+
+for i in range(360):
+    shm_data.lidar[i] = 65535
+
+input(
+    f"""
+Shared memory key = {shm_ptr.key}
+
+Start the native application the gdb.
+In the RIOT shell:
+    - use the key with '_set_shmem_key,
```

Comment:
missing '

Reply:
Done.
File moved to 'examples/lidar_obstacles'.